controllers/configmap_controller.go

```diff
@@ -216,10 +230,50 @@ func (r *ConfigMapReconciler) reconcileServices(ctx context.Context, windowsServ
 			kubeTypes.NamespacedName{Namespace: r.watchNamespace, Name: windowsServices.Name}, "Error", err.Error())
 		return nil
 	}
+	r.setValues(node, data)
 	// TODO: actually react to changes to the services ConfigMap
 	return nil
 }
 
+func (r *ConfigMapReconciler) setValues(node *core.Node, data *servicescm.Data) {
+	// machineconfig.Spec.Config
+	// unmarshal the json, and set the data to the machineconfig's value
+
+	// check the node.annotations and node.label
+
+	for i := range data.Services {
```

Comment:
This can be simplified, consider this:
```
for _, svc := range data.Services {
		if svc.Name == "kubelet" {
			if strings.Contains(svc.Command, "--node-labels"){
				
			}
		}
	}

```

Reply:
I actually can't use a for x in range y for this loop because for some reason if you do that, it edits a copy instead of the original object! It's a weird go quirk that you have to use the index if you want to edit the thing the loop is referencing. However, strings.Contains would totally be way better lol